HIP/hip-1021.md

```diff
@@ -0,0 +1,65 @@
+---
+hip: 1021
+title: Improve Assignment of Auto-Renew Account ID for Topics
+author: Michael Kantor (@kantorcodes)
+type: Standards Track
+category: Service
+status: Draft
+created: 2024-08-01
+requested-by: TierBot
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/pull/1021
+needs-council-approval: Yes
+---
+
+## Abstract
+
+This HIP proposes a modification to the Hedera Consensus Service (HCS) to:
+ - Enable setting the `autoRenewAccountId` to the account creating a `TopicCreateTransaction` when an Admin Key is not present during Topic Creation.
+ - It ensures that for all existing topics with only a Submit Key and no `autoRenewAccountId`, the original `payer_account_id` of the topic is retroactively set as the `autoRenewAccountId`.
```

Comment:
We should not modify existing topics IMO:

- We usually do not modify existing entities. A user may have left autoRenewAccount empty intentionally and disagrees with this change.
- This would have to be initiated outside of the consensus node because the consensus node does not store the original `payer_account_id`.
- It is unclear who should pay for these transactions.